### PR TITLE
chore: Point to correct react native docs in getting started docs

### PR DESCRIPTION
**Pull request by olerichter00**
### Description

Point to the correct React Native documentation in the getting started docs. The docs currently link to "Start a new React Native project with Expo." I suppose older documentation is needed because we don't use Expo.

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f639ef36357651506234dca88eaed9c8fbcdfdaf
</p>

File: docs/getting_started.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Check that Command Line Tools version is added in the Locations tab. Xcode>Setti
 >
 > `export JAVA_HOME="<YOUR-ASDF-JAVA-PATH-HERE>"`
 
-1. Android development environment: Follow the [official docs](https://reactnative.dev/docs/environment-setup). Select "React Native CLI Quickstart" tab
+1. Android development environment: Follow the [official docs](https://reactnative.dev/docs/0.70/environment-setup?guide=native). Select "React Native CLI Quickstart" tab
```
Comment (brainbicycle) [inline docs/getting_started.md:50]: think we should point to 0.73 docs: https://reactnative.dev/docs/0.73/environment-setup?package-manager=yarn

Comment (olerichter00) [reply docs/getting_started.md]: Thanks! I've updated the link 🙏

**Review by brainbicycle — APPROVED** [review]
thanks!

**Pull request merged**